bower.json

```diff
@@ -27,10 +27,23 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-ace": "^4.0.0",
-    "purescript-halogen": "^3.1.1",
-    "purescript-now": "^3.0.0",
-    "purescript-random": "^3.0.0",
-    "purescript-refs": "^3.0.0"
+    "purescript-ace": "^7.0.0",
+    "purescript-halogen": "4.0.0",
+    "purescript-now": "^4.0.0",
+    "purescript-random": "^4.0.0",
+    "purescript-refs": "^4.1.0",
+    "purescript-aff": "^5.1.1",
+    "purescript-foreign-object": "^2.0.1",
+    "purescript-prelude": "^4.1.1"
+  },
+  "resolutions": {
+    "purescript-foreign-object": "^2.0.1",
+    "purescript-typelevel-prelude": "^4.0.0",
+    "purescript-web-html": "^2.0.0",
+    "purescript-web-uievents": "^2.0.0",
+    "purescript-web-file": "^2.0.0",
+    "purescript-web-dom": "^2.0.0",
+    "purescript-web-storage": "^3.0.0",
+    "purescript-web-events": "^2.0.0"
```

Comment:
I think something is wrong somewhere if you were forced to make resolutions - maybe some other dependency or dependencies that need updating first - I don't mind making resolutions in projects / apps that aren't going to be depended upon, but having it in a library is going to cause issues for anyone who wants to rely on it too.

Reply:
I hate bower so much. ok I can take a look, I just updated everything to latest

Reply:
I use `node-check-updates` to bump everything to the latest version (you can use a flag to run it on `bower` too) if you don't already know about that. It doesn't help with figuring out where the conflicts are when bower has a meltdown unfortunately, but it's useful to take some of the pain out!